docs/desktop/gnome/dconf.md

```diff
@@ -1,75 +1,73 @@
 ---
 title: dconf Config Editor
 author: Ezequiel Bruni
-contributors: Steven Spencer
+contributors: Steven Spencer, Ganna Zhyrnova
 ---
 
 ## Introduction
 
-GNOME takes a very streamlined approach to its user interface and its features. This is not a bad thing, as it is easy enough to learn, and the default GNOME experience lets you get right to work doing your work.
+GNOME takes a very streamlined approach to its user interface and its features. This is not bad, as it is easy to learn, and the default GNOME experience lets you get to work doing your work.
 
-However, this approach also means that it is not as easily configurable as some people would like. Sure, if you can not find what you need in the Settings panel, you can install GNOME Tweaks to expand your options. You can even install GNOME extensions to get new features and options.
+However, this approach also means it is not as quickly configurable. If you can not find what you need in the Settings panel, you can install GNOME Tweaks to expand your options. You can even install GNOME extensions to get new features and options.
 
-But what if you want to see all the little settings, features, and configurations that the GNOME developers have hidden from you? Sure, you can look up your current problem online, and type in a command to change an obscure variable, or you could install `dconf Editor`.
+But what if you want to see all the little settings, features, and configurations the GNOME developers have hidden? You can look up your current problem online and type in a command to change an obscure variable, or you can install `dconf Editor`.
 
-`dconf Editor` is essentially a GNOME Settings app that has *everything*. In fact, it might remind you a little bit of the Windows Registry, and it *is* similar. However, it is more readable, it only covers GNOME functionality, and some of the software built for GNOME.
+`dconf Editor` is a GNOME Settings app with *everything*. It might remind you a little bit of the Windows Registry, and it *is* similar. However, it is more readable; it only covers GNOME functionality, and some software is built for GNOME.
```

Comment:
```suggestion
`dconf Editor` is a GNOME Settings app with *everything*. It might remind you a little bit of the Windows Registry, and it *is* similar. However, it is more readable; it only covers GNOME functionality and some software built for GNOME.
```

Reply:
?
it covers some software build for GNOME

Reply:
Hey @SergeCroise You aren't wrong about how this sentence reads, but I also did not like the semi-colon. I did a quick edit.